.github/workflows/codeql-analysis-noop.yml

```diff
@@ -0,0 +1,34 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [main, "*"]
+    paths:
+      - "**/*.yml"
```

Comment:
If more paths are added to `paths-ignore` I assume they need to be added here as well? Maybe adding a comment to the other file would be good.

Reply:
Nice, if-this-then-that :)